src/game/common/gamelod.cpp

```diff
@@ -345,10 +347,74 @@ bool GameLODManager::Set_Static_LOD_Level(StaticGameLODLevel level)
 
 void GameLODManager::Apply_Static_LOD_Level(StaticGameLODLevel level)
 {
-#ifdef GAME_DLL
-    // TODO requires parts of TerrainVisual and GameClient classes.
-    Call_Method<void, GameLODManager, StaticGameLODLevel>(PICK_ADDRESS(0x0047AC60, 0x007E40D9), this, level);
-#endif
+    StaticGameLOD lod;
+
+    if (m_staticLODLevel != -1) {
+        lod = m_staticLOD[m_staticLODLevel];
+    }
+
+    if (level == STATLOD_CUSTOM) {
+        Refresh_Custom_Static_LOD();
+    }
+
+    StaticGameLOD *new_lod = &m_staticLOD[level];
```

Comment:
I suggest make this reference instead for simplicity
```c++
StaticGameLOD &new_lod = m_staticLOD[level];
```

and then just access with dot
```c++
        g_theWriteableGlobalData->m_maxParticleCount = new_lod.max_particle_count;
        g_theWriteableGlobalData->m_shadowVolumes = new_lod.use_shadow_volumes;
        g_theWriteableGlobalData->m_shadowDecals = new_lod.use_shadow_decals;
```